sheepdog/api.py

```diff
@@ -152,10 +152,6 @@ def app_init(app):
     db_init(app)
     # exclude es init as it's not used yet
     # es_init(app)
-    try:
-        app.secret_key = app.config["FLASK_SECRET_KEY"]
```

Comment:
why are we removing this? Flask requires this to read/write session iirc.

Reply:
We were removing settings from GDC, that we could not find references to in code. 

Interesting that none of the unit/ integration tests cover this if this is managing sessions 🤔  

Happy to put this back in if you think if we still need it.

Reply:
It's used internally by Flask. We should not remove it. https://flask.palletsprojects.com/en/stable/quickstart/#sessions

It's _possible_ that sheepdog doesn't use any Flask session stuff right now, but I would rather just leave this in so it's already configured if someone uses it in the future. And if there is use of this, if we remove it, the sessions will all be unreadable